plugins/scaffolder-backend-module-sentry/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@backstage/plugin-scaffolder-backend-module-sentry",
+  "version": "0.1.0",
```

Comment:
I'd start this at `0.0.0` and then the changeset as minor will publish `0.1.0` :)

Reply:
done